src/core.js

```diff
@@ -1775,6 +1775,50 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
         /* eslint-disable-next-line no-continue */
         continue;
 
+      } else if (i === 'className') {
```

Comment:
It's there any reason to not use `switch` instead of `if ... else if ... else if ... else if ... else if ... else if ....`?

Reply:
Good question. 
Earlier was only a few options, but after added two new `switch` is a better choice.

Reply:
Unfortunately, our `eslint` rules don't  allow `switch` in `for in` loop - `The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.eslint(guard-for-in)`

Reply:
Unfortunately, our `eslint` rules don't allow `switch` in `for in` loop - `The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.eslint(guard-for-in)`.

Reply:
So, why not to iterate over `Object.keys(settings).filter(...)`  instead?